src/runtime/msg/action.rs

```diff
@@ -143,6 +143,13 @@ pub enum ActionPlayer {
     PausedChanged {
         paused: bool,
     },
+    /// Play next video, if there is one, applicable to e.g.
+    /// movie series and playing the next episode.
+    NextVideo,
+    /// Video player has ended.
+    /// 2 scenarios are possible:
+    /// - We've watched a movie to the last second
+    /// - We've watched a movie series to the last episode and the last second
```

Comment:
Not sure if it indicates that we've watched to the last episode. This just indicates that the end of currently playing stream was reached.

Reply:
Good point, will check with @tymmesyde as well